components/Mermaid/index.tsx

```diff
@@ -10,35 +10,34 @@ let mermaid: MermaidType;
 
 const Mermaid = ({ children }: React.PropsWithChildren) => {
   const theme = useContext(ThemeContext);
-  const containerRef = useRef<HTMLDivElement>(null);
+  const [content, setContent] = useState('');
 
   useEffect(() => {
-    if (typeof window !== 'undefined') {
-      import('mermaid').then(async module => {
+    if (typeof window === 'undefined') return;
+
+    // Initialize Mermaid
+    import('mermaid').then(module => {
+      if (!mermaid) {
         mermaid = module.default;
-        // Initialize Mermaid with the current theme
         mermaid.initialize({
+          startOnLoad: false,
```

Comment:
the default is true but we probably want it to be false https://mermaid.js.org/config/schema-docs/config.html#startonload